code/modules/mob/living/carbon/xenomorph/XenoUpgrade.dm

```diff
@@ -158,7 +158,7 @@ Queen		 0.0	 0.1	 0.2	 0.3
 					spit_delay = 10
 					upgrade_threshold = 400
 					caste_desc = "A ranged damage dealer. It looks a little more dangerous."
-					spit_types = list(/datum/ammo/xeno/toxin/medium/upgrade1, /datum/ammo/xeno/acid/medium)
+					spit_types = list(/datum/ammo/xeno/acid/heavy/spitter)
```

Comment:
This looks very convoluted. Why do spitters get their own version of heavy?

Reply:
Alter the plasma pool and regeneration logic then.

Reply:
I think it's pretty obvious that it's better to give them a custom projectile than to alter these values, including all the upgrades.

Reply:
Let's keep ammo simple.